tests/System.IO.Compression.Tests/System.IO.Compression.Tests.csproj

```diff
@@ -22,7 +22,7 @@
     <ProjectReference Include="..\..\src\System.Buffers.Primitives\System.Buffers.Primitives.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <Content Include="BrotliTestData\*">
+    <Content Include="BrotliTestData\**">
```

Comment:
I am curious, is this how wildcard works? What was the behavior before this change?